platform/qemu.go

```diff
@@ -214,3 +219,102 @@ func mkpath(basedir string) (string, error) {
 	defer f.Close()
 	return f.Name(), nil
 }
+
+func CreateQEMUCommand(board, uuid, biosImage, consolePath, confPath, diskImagePath string, isIgnition bool, options MachineOptions) ([]string, []*os.File, error) {
+	var qmCmd []string
+	combo := runtime.GOARCH + "--" + board
+	switch combo {
+	case "amd64--amd64-usr":
```

Comment:
Aside: ahh right, we'll eventually need to expand this list for more archs, which... yeah would be good to share this with `cosa run`. Maybe for now a comment here to that effect?

Reply:
Isn't `cosa run` libvirt based? That level of abstraction should eliminate a lot of the need for this native arch + intended  arch matrix.

Reply:
No, it uses `qemu-kvm` directly, see: https://github.com/coreos/coreos-assembler/blob/f42b840cfe1b4c3d564b1c33b91768222435da8f/src/cmd-run#L181. We already have some minor arch-specific optioneering going on there.

Reply:
Ah, TIL, will add a comment.